docker-compose.yaml

```diff
@@ -56,3 +56,20 @@ services:
       - LDAP_ADMIN_PASSWORD=adminpassword
       - LDAP_USERS=alice,bob,foo
       - LDAP_PASSWORDS=password1,password2,password3
+
+  # this container will take at least 20 seconds for the ad service to be
+  # available so manual test runs should take this into account
+  ad:
```

Comment:
Curious if we see benefit to adding the container to CI rn if it isn't being used. If it's meant to be local-only for now I wonder if we should save the overhead of creating this container which can take upwards of 20s? Can always add back in once we figure out how rotate root tests fit into CI